docs/specs/preloadDesktopAgents.md

```diff
@@ -0,0 +1,41 @@
+# Preload DA Specification
```

Comment:
Does this not already exist somewhere?

Reply:
Sort of, it's whatever is in the overview document - which is trying to speak to both Application developers and Platform developers (DA developers) at the same time. Consequently it doesn't do a great job of either as they need different content.

If we broke out specifications for DA's like this then I think we could tweak the overview doc to be clearer by splitting into functional goals, specs for using the DA API and specs for implementing it.